en/docs/secure-health-apis/guides/getting-patientid.md

```diff
@@ -0,0 +1,96 @@
+# Retrieving the Patient ID
+
+In this example we will walk through the steps that how we can get the Patient ID by executing the SMART on FHIR flow in Healthcare Accelerator. 
+
+## Prerequisites
+1. Intall the latest WSO2 API Manager version and install the [WSO2 Healthcare Accelerator](../../install-and-setup/manual.md).
+2. Start the WSO2 API Manager server by navigating to <APIM_HOME>/bin directory and executing this command. 
+```
+./api-manager.sh
+```
+
+## Step 1 - Create the Application 
+1. Navigate to Developer Portal in your web browser: `https://localhost:9443/devportal/`
+2. Self signup a user by clicking 'SIGN-IN' button, it will redirect you to the Authentication Portal. 
+3. Click on 'Create Account' link, provide a username and click on 'Proceed to Self Register'. 
+4. Fill in the details and create the account. 
+5. Log into Devportal with the crendetials of the above user. 
+6. Click on 'ADD NEW APPLICATION' button, provide application details and save the application. 
+7. Go to created application, and click on 'Production Keys' in left side panel. 
+8. For Grant Types, enable 'Code' grant type, provide the Callback URL. This is the app that we need to redirect. For the timebeing let's provide `http://localhost:3000`
+9. Click on 'GEENRATE KEYS', and click on 'UPDATE' button. 
+10. Note down the Consumer Key and Consumer Secret. 
```

Comment:
If we have already steps in the apim docs we should link to it and say to follow the steps. Also add screenshots where necessary

Reply:
Fixed in https://github.com/wso2/open-healthcare-docs/pull/44/commits/265eb2611a3f4cda6c4e230c05fb3f442d5afcee